contentcuration/contentcuration/utils/minio_utils.py

```diff
@@ -94,21 +108,24 @@ def ensure_bucket_deleted(bucket=None):
     # skip bucket creation there and just bug Aron to create buckets with
     # public-read access for you
     if "storage.googleapis.com" in host:
```

Comment:
We should probably update this check to use the same utility as the above function: `is_gcs_backend()`